packages/manager/modules/bm-server-components/src/general-information/installation/choice/server-installation-choice.html

```diff
@@ -39,17 +39,26 @@
             ></span
         ></oui-radio-label>
     </oui-radio>
-
-    <oui-radio
-        data-ng-if="$ctrl.isByoiAvailable"
-        name="choiceInstall"
-        data-model="$ctrl.choice.value"
-        data-value="$ctrl.choice.image"
+    <div
+        class="alert alert-info mt-4"
+        role="alert"
     >
-        <oui-radio-label>
+        <span
+            data-ng-bind-html="'server_configuration_installation_custom_image_info' | translate:{ t0: ('server_configuration_installation_ovh_desktop_CUSTOMER' | translate), t1: ('server_configuration_installation_ovh_step1_desktop' | translate) }"
```

Comment:
@anooparveti then since you wanted me to remove the bold html tag, we don't need data-ng-bind-html anymore, I can just use data-ng-translate

Reply:
Sorry I misread your comment... Adding the bold tags around the variable